PureInvoke/Functions/Invoke-AdvapiLookupAccountSid.ps1

```diff
@@ -0,0 +1,69 @@
+
+function Invoke-AdvApiLookupAccountSid
+{
+    <#
+    .SYNOPSIS
+    Calls the Advanced Windows 32 Base API (advapi32.dll) `LookupAccountSid` function.
+
+    .DESCRIPTION
+    The `Invoke-AdvApiLookupAccountSid` function calls the advapi32.dll API's `LookupAccountSid` function, which looks up a
+    SID and returns its account name, domain name, and use. Pass the SID as a byte array to the `Sid` parameter and the
+    system name to the `SystemName` parameter, which are passed to `LookupAccountSid` as the `Sid` and `lpSystemName`
+    arguments, respectively. The function returns an object with properties for each of the `LookupAccountSid`
+    function's out parameters: `Name`, `ReferencedDomainName`, and `Use`.
+
+    .LINK
+    https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-lookupaccountsida
+
+    .EXAMPLE
+    Invoke-AdvApiLookupAccountSid -Sid $sid
+
+    Demonstrates how to call this function by passing a sid to the `Sid` parameter.
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [byte[]] $Sid,
+
+        [String] $SystemName

```

Comment:
`$SystemName` should be `$ComputerName` for consistency with other functions in this module.